lnbits/wallets/clightning.py

```diff
@@ -84,9 +86,17 @@ async def create_invoice(
             error_message = f"lightningd '{exc.method}' failed with '{exc.error}'."
             return InvoiceResponse(False, label, None, error_message)
 
-    async def pay_invoice(self, bolt11: str) -> PaymentResponse:
+    async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
+        invoice = lnbits_bolt11.decode(bolt11)
+        fee_limit_percent = fee_limit_msat / invoice.amount_msat * 100
+
+        payload = {
+            "bolt11" : bolt11,
+            "maxfeepercent" : fee_limit_percent
```

Comment:
Doesn't this need to be a string? Precision might cause issues.
```suggestion
            "maxfeepercent" : "{:.1f}".format(fee_limit_percent)
```

Reply:
I did some tests with various precision (out of curiosity) and didn't induce any issue. But since fee_limit_percent is a float now, maybe it's better to have control over the precision.

Reply:
Maybe I would also revert this change, it seems to me that clightning handles well many digits precision... Bad idea?

![Snímek obrazovky pořízený 2021-12-03 10-33-45](https://user-images.githubusercontent.com/80261724/144580487-8fa7df08-283e-4eda-b9cb-679d4a6c689d.png)

Reply:
Or `"maxfeepercent" : "{:.11f}".format(fee_limit_percent)` to be on the safe side?

Reply:
We should set the precision ourselves, the more precise the better. 11 digits seems high, but why not. If you've tested it (I can't), it should be fine :)